src/query/xasl_cache.c

```diff
@@ -1173,6 +1198,7 @@ xcache_unfix (THREAD_ENTRY * thread_p, XASL_CACHE_ENTRY * xcache_entry)
     }
 }
 
+
 /*
```

Comment:
```suggestion

/*
```
해당 PR과 관련없는 newline 인것 같아서 남깁니다

Reply:
적용하겠습니다.